worker/include/RTC/SCTP/chunks/ShutdownChunk.hpp

```diff
@@ -0,0 +1,106 @@
+#ifndef MS_RTC_SCTP_SHUTDOWN_CHUNK_HPP
+#define MS_RTC_SCTP_SHUTDOWN_CHUNK_HPP
+
+#include "common.hpp"
+#include "Utils.hpp"
+#include "RTC/SCTP/Chunk.hpp"
+
+namespace RTC
+{
+	namespace SCTP
+	{
+		/**
+		 * SCTP Shutdown Association Chunk (SHUTDOWN) (7).
+		 *
+		 * @see RFC 9260.
+		 *
+		 *  0                   1                   2                   3
+		 *  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+		 * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+		 * |   Type = 7    |  Chunk Flags  |          Length = 8           |
+		 * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+		 * |                      Cumulative TSN Ack                       |
+		 * +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+		 *
+		 * - Chunk Type (8 bits): 7.
+		 * - Flags (8 bits): All set to 0.
+		 * - Length (16 bits): 8.
+		 * - Cumulative TSN Ack (32 bits): The largest TSN, such that all TSNs
+		 *   smaller than or equal to it have been received and the next one has
+		 *   not been received.
+		 */
+
+		// Forward declaration.
+		class Packet;
+
+		class ShutdownChunk : public Chunk
+		{
+			// We need that Packet calls protected and private methods in this class.
+			friend class Packet;
+
+		public:
+			static const size_t ShutdownChunkHeaderLength{ 8 };
```

Comment:
Considering we are inside `ShutdownChunk` class, this could be named `HeaderLength` as it's going to be accessed as `ShutdownChunk::HeaderLength`.

Reply:
If I forget to re-declare `HeaderLength` in a `FooChunk` class whose header length is not the default one and then I use `FooChunk::HeaderLength` is will give me the value of `Chunk::HeaderLength` which is 4 and things would fail, so I prefer to be explicit and prefix everything with the full subclass name.